tests/test_filter.py

```diff
@@ -408,6 +430,50 @@ def test_application_to_dataset():
         filter.apply(dataset, ["yy", "x"])
 
 
+@pytest.mark.parametrize(
+    "filter_args",
+    [
+        dict(
+            filter_scale=8.0,
+            dx_min=1.0,
+            n_steps=0,
+            filter_shape=FilterShape.GAUSSIAN,
+            n_iterations=1,
+        )
+    ],
+)
+def test_iterated_filter(grid_type_and_input_ds, filter_args):
+    "Test that the iterated Gaussian filter gives a result close to the original"
+
+    grid_type, da, grid_vars = grid_type_and_input_ds
+
+    iterated_filter_args = filter_args.copy()
+    iterated_filter_args["n_iterations"] = 2
```

Comment:
Should we add another iterated filter with `n_iterations > 2`? This is optional.

Reply:
To test more iterations, we easily could add an additional `@pytest.mark.paratmetrize('n_iterations', [2, 3, 4]))` decorator to the function. This seems like a good idea to me.